src/plugins/GUITestBase/src/tests/common_scenarios/pcr_primer_design_for_dna_assembly/GTTestsPcrPrimerDesignForDnaAssembly.cpp

```diff
@@ -298,6 +298,45 @@ GUI_TEST_CLASS_DEFINITION(test_0007) {
                                                                                   .arg(currentEnd))
 }
 
+GUI_TEST_CLASS_DEFINITION(test_0008) {
+    // Open common_data/pcr_primer_design/gfp.fa.
+    GTUtilsProject::openFileExpectSequence(os, testDir + "_common_data/pcr_primer_design/gfp.fa", "gfp");
+    // Open the PCR Primer Design tab.
+    GTUtilsPcrPrimerDesign::openTab(os);
+    QWidget *sequence = GTUtilsSequenceView::getActiveSequenceViewWindow(os);
```

Comment:
nit: use "auto sequenceWidget" instead of "Widget sequence"
This way the rest of the code will be more readable.